_additional/fakeClient/fakeClient.js

```diff
@@ -0,0 +1,23 @@
+const _ = require('lodash');
+
+const ClientHandler = require('./lib/ClientHandler');
+const FileReader = require('./lib/FileReader');
```

Comment:
zaimportuj może samą funkcję:P Bo potem to FileReader.read wygląda jak w Javie jakiejś:P

Reply:
Argument z Javą mnie przekonał.